src/transformState.js

```diff
@@ -6,6 +6,26 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const dia of actions) {
```

Comment:
what's `dia`? it's an array of `actions`, please use corresponding name - `action`